tofu/gcp/tenant/resources/k8s/falkordb-deployment/main.tf

```diff
@@ -1,161 +1,63 @@
 locals {
```

Comment:
so now we deploy 1 tenant for each type?
need to add test the check that all are working correctly

Reply:
Yes, the tenant chooses which option he wants.
And of course, the testing should include one tenant for each type.
We must also test that the deployment is being split/grouped according to the desire option.